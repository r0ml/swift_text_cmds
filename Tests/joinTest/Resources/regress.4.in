Jean Henri Dunant,Switzerland
Frederic Passy,France
Elie Ducommun,Switzerland
Baroness Bertha Sophie Felicita Von Suttner
