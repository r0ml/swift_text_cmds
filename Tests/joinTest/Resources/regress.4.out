1901,Switzerland
1901,France
1902,Switzerland
1905,<<NULL>>
